src/DrugstoreApp/Backend/DrugstoreAPI/Drugstore/Repository/Interfaces/IFeedbackRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Drugstore.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Drugstore.Repository.Interfaces
+{
+    public interface IFeedbackRepository
+    {
+        public bool Delete(int id);
+        public List<Feedback> GetAll();
+        public bool Save(Feedback newFeedback);
```

Comment:
Save method should return Feedback object created in db.

Reply:
I will use Command-Query separation pattern, so it's gonna be void!